db/migrate/20220923185647_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
+  end
```

Comment:
- [ ] As per the given ERD, the reference name should be `author` instead of `user`, which should be similar to:

```ruby
...
  def change
    add_reference :posts, :author, references: :users, null: false, foreign_key: {to_table: :users}
  end
...